Web/faith_in_humanity_score.6h.py

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/python
-# coding=utf-8
-#
-# <xbar.title>Faith in humanity score</xbar.title>
-# <xbar.version>v1.0</xbar.version>
-# <xbar.author>impshum</xbar.author>
-# <xbar.author.github>impshum</xbar.author.github>
-# <xbar.desc>Displays current score from https://faithinhumanityscore.com</xbar.desc>
-#
-# by impshum
-
-from requests import get
-
-data = get('https://faithinhumanityscore.com/api.php').json()
```

Comment:
Looks like there might be a way to salvage this plugin.

@impshum Any interest in updating this?

Reply:
Fixed it: https://faithinhumanityscore.com/api.php